server/index.js

```diff
@@ -28,46 +75,46 @@ app.get('/', (req, res) => {
   res.sendFile(path.join(`${__dirname}/../react-client/src/index.html`));
 });
 // CATEGORY GET '/api/categories': List of all categories.
-app.get('/api/categories', (req, res) => {
+app.get('/api/categories', isLoggedIn, (req, res) => {
```

Comment:
Confused on this for the category requests- do we need the "isLoggedIn"?

Reply:
really just depends on if what we want people to have access to and not have access to.